### DISCUSSION TITLE
ANE NetworkInfo

### DISCUSSION THREAD

**Discussion by skramovskiy** [airsdk/Adobe-Runtime-Support#3223] [General]
Good afternoon
I use ANE NetworkInfo to scan wifi networks 

It works fine on Android
On the desktop too
On iOS, the IP determines cellular networks, but the Wi-Fi does not. 
What could be the problem? 
What permissions do I need to register for ios?

**Comment by marchbold** [comment]
Can you elaborate some more on exactly what you are doing? I assuming you are using distriqt's `com.distriqt.NetworkInfo` extension?

**Reply by skramovskiy** [reply]
yes, that's right. I use this library on Android - and everything works correctly. But when launched on an iPhone, not all functionality works. if we are connected to a cellular network, it determines the IP address. But if to wifi, then the iPhone does not detect the current ip on the network

**Reply by marchbold** [reply]
Okay, could you log an issue with our repo and I'll get someone to look into it: https://github.com/distriqt/ANE-NetworkInfo/issues/new

Make sure you include the versions of iOS that you have experienced this issue on. Cheers
